diff -aurN celestia-1.6.0.orig/src/celestia/gtk/data/celestia.desktop celestia-1.6.0/src/celestia/gtk/data/celestia.desktop
--- celestia-1.6.0.orig/src/celestia/gtk/data/celestia.desktop	2009-08-13 08:16:29.000000000 +0200
+++ celestia-1.6.0/src/celestia/gtk/data/celestia.desktop	2009-08-13 08:20:42.000000000 +0200
@@ -1,12 +1,10 @@
 [Desktop Entry]
-Encoding=UTF-8
-Version=1.4.0
 Type=Application
 Exec=celestia
 Icon=celestia
-X-GNOME-DocPath=
 Terminal=false
 Name=Celestia
 Comment=Space Simulator
 StartupNotify=true
-Categories=Astronomy;Science;Application;
+Categories=GTK;Astronomy;Education;Science;
+
